pkg/client/fake/client.go

```diff
@@ -86,23 +85,6 @@ const (
 	maxGeneratedNameLength = maxNameLength - randomLength
 )
 
-// NewFakeClient creates a new fake client for testing.
-// You can choose to initialize it with a slice of runtime.Object.
-//
-// Deprecated: Please use NewClientBuilder instead.
-func NewFakeClient(initObjs ...runtime.Object) client.WithWatch {
```

Comment:
Urgh, I'd really like to keep this one, its more convenient than the builder for simple cases

Reply:
Sure. I'll drop the deprecation